tests/test_metadata.py

```diff
@@ -33,51 +31,103 @@
 [project.urls]
 Homepage = "https://github.com/pypa/sampleproject"
 "Bug Tracker" = "https://github.com/pypa/sampleproject/issues"
-
-[tool.hatch.metadata.hooks.requirements_txt]
-filename = "requirements.txt"
 """
 
 
-def test_wheel(tmp_pathplus: PathPlus):
+def get_pkginfo(tmp_pathplus: PathPlus, build_func: Callable,
+				pyproject_toml: str) -> Union[pkginfo.SDist, pkginfo.Wheel]:
```

Comment:
```suggestion
def get_pkginfo(
		tmp_pathplus: PathPlus,
		build_func: Callable,
		pyproject_toml: str,
		) -> Union[pkginfo.SDist, pkginfo.Wheel]: 
```

That should make both formate and flake8 happy.